Dockerfile

```diff
@@ -95,6 +96,16 @@ RUN curl -sSL https://github.com/grpc/grpc-web/releases/download/${grpc_web_vers
     -o /tmp/grpc_web_plugin && \
     chmod +x /tmp/grpc_web_plugin
 
+FROM swift:$swift_version AS build-swift
+
+ARG grpc_swift_version
+
+WORKDIR /tmp
+RUN git clone --depth 1 -b $grpc_swift_version https://github.com/grpc/grpc-swift.git
```

Comment:
It seems as though Apple has an official plugin for this now: https://github.com/apple/swift-protobuf

Reply:
It's not quite the same thing. `swift-protobuf` is a package that can [de]serialize protobuf-encoded messages. `grpc-swift` is the _grpc_ runtime, allowing to implement grpc clients and servers in Swift.